apps/Learning-Buddy/screens/NewQuizScreen.jsx

```diff
@@ -0,0 +1,354 @@
+              />
+              <BigButton
+                buttonColor={colors.lightGrey}
+                textColor={
+                  selectedQuestionType === 'True/False'
+                    ? colors.white
+                    : colors.black
+                }
+                content={'True/False'}
+                onPress={() => handleQuestionTypePress('True/False')}
+              />
+              <BigButton
+                buttonColor={colors.lightGrey}
+                textColor={
+                  selectedQuestionType === 'Completion'
+                    ? colors.white
+                    : colors.black
+                }
+                content={'Completion'}
+                onPress={() => handleQuestionTypePress('Completion')}
+              />
+              <BigButton
+                buttonColor={colors.lightGrey}
+                textColor={
+                  selectedQuestionType === 'Matching'
+                    ? colors.white
+                    : colors.black
+                }
+                content={'Matching'}
+                onPress={() => handleQuestionTypePress('Matching')}
+              />
+              <BigButton
+                buttonColor={colors.lightGrey}
+                textColor={
+                  selectedQuestionType === 'Essays'
+                    ? colors.white
+                    : colors.black
+                }
+                content={'Essays'}
+                onPress={() => handleQuestionTypePress('Essays')}
+              />
+            </View>
+          </View>
+          <View>
+            <Text style={localStyles.title}>Difficulty Level</Text>
+            <View style={localStyles.container}>
+              <BigButton
+                buttonColor={colors.lightGrey}
+                textColor={
+                  selectedDifficulty === 'Beginner'
+                    ? colors.white
+                    : colors.black
+                }
+                content={'Beginner'}
+                onPress={() => handleDifficultyPress('Beginner')}
+              />
+              <BigButton
+                buttonColor={colors.lightGrey}
+                textColor={
+                  selectedDifficulty === 'Intermediate'
+                    ? colors.white
+                    : colors.black
+                }
+                content={'Intermediate'}
+                onPress={() => handleDifficultyPress('Intermediate')}
+                disabled={true}
+              />
+              <BigButton
+                buttonColor={colors.lightGrey}
+                textColor={
+                  selectedDifficulty === 'Advanced'
+                    ? colors.white
+                    : colors.black
+                }
+                content={'Advanced'}
+                onPress={() => handleDifficultyPress('Advanced')}
+                disabled={true}
+              />
+            </View>
+          </View>
+          <View>
```

Comment:
If we have no functionality for this section, it should be commented out to avoid confusion when demoing/ sharing with product.

Reply:
I wanted to tie this with the wireframe for demonstration purposes, which is why it is not commented out.